Moving Clay app to production - ImportError: No module named wsgi

Hi,

First, I want to say thanks for open sourcing Clay and @synack for helping me out earlier. Thought I'd try submitting a question here.

I'm trying to move my local Clay app to production, specifically to Heroku. 

When I run this,
gunicorn clay.wsgi:application

I get this error:
2014-03-17 19:11:54 [30703] [ERROR] Exception in worker process:
Traceback (most recent call last):
  File "/home/lamr/uber-clay/venv/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 495, in spawn_worker
    worker.init_process()
  File "/home/lamr/uber-clay/venv/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 106, in init_process
    self.wsgi = self.app.wsgi()
  File "/home/lamr/uber-clay/venv/local/lib/python2.7/site-packages/gunicorn/app/base.py", line 114, in wsgi
    self.callable = self.load()
  File "/home/lamr/uber-clay/venv/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 62, in load
    return self.load_wsgiapp()
  File "/home/lamr/uber-clay/venv/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 49, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/lamr/uber-clay/venv/local/lib/python2.7/site-packages/gunicorn/util.py", line 354, in import_app
    **import**(module)
ImportError: No module named wsgi
Traceback (most recent call last):
  File "/home/lamr/uber-clay/venv/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 495, in spawn_worker
    worker.init_process()
  File "/home/lamr/uber-clay/venv/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 106, in init_process
    self.wsgi = self.app.wsgi()
  File "/home/lamr/uber-clay/venv/local/lib/python2.7/site-packages/gunicorn/app/base.py", line 114, in wsgi
    self.callable = self.load()
  File "/home/lamr/uber-clay/venv/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 62, in load
    return self.load_wsgiapp()
  File "/home/lamr/uber-clay/venv/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 49, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/lamr/uber-clay/venv/local/lib/python2.7/site-packages/gunicorn/util.py", line 354, in import_app
    **import**(module)
ImportError: No module named wsgi

I'm guessing it's because I dont have a clay.wsgi file.

I found this for flask
http://flask.pocoo.org/docs/deploying/mod_wsgi/#creating-a-wsgi-file

but creating a clay.wsgi with this inside 
"from clay import app as application"
didn't change the error code.

Does anyone have experience moving Clay to production?
Thanks for your help!

--- a/README.md
+++ b/README.md
@@ -3,7 +3,6 @@ Clay is a framework for building RESTful backend services using best practices.
 It's a wrapper around Flask (http://flask.pocoo.org) along with several convenient, standardized, performance enhanced ways of performing common tasks like sending email and connecting to a database.
 
 Clay is available under the MIT License.
-Documentation is available at http://uber.github.com/clay/
 
 Git repository: https://github.com/uber/clay.git
 Install via PyPI: pip install clay-flask

